src/git_config.rs

```diff
@@ -42,18 +44,26 @@ impl GitConfig {
             config: git2::Config::open(path).unwrap(),
             repo: None,
             enabled: true,
+            ignored_keys: vec![],
         }
     }
 
     pub fn get<T>(&self, key: &str) -> Option<T>
     where
         T: GitConfigGet,
     {
-        if self.enabled {
-            T::git_config_get(key, self)
-        } else {
-            None
+        if !self.enabled {
+            return None;
         }
+
+        if !self.ignored_keys.is_empty() {
+            let key_name = &key.rsplit('.').collect::<Vec<&str>>()[0];
```

Comment:
keys will be 

1. core.pager
2. delta.line-numbers
3. delta.features.line-numbers

I think users don't want to specify parent key, so I split key here.